### ISSUE TITLE
Instance / NPC: Ulduar / Flame Leviathan 

### ISSUE THREAD

**Issue by Crispybuilt** [dalaranwow/dalaran-wow#523]
Instance - Ulduar (ID:4273) - (10) & (25) man mode
NPC - Flame Leviathan  33113/ 34003

- **Issue:** *Once all the towers are destroyed to attend Flame Leviathan (easy mode), if you wipe on him and attend again, he will still re-gain all 4 buffs from towers despite they were destroyed*.

**How it should work:** Flame Leviathan will lose tower buffs once they are destroyed and not regain a tower buff if that tower is destroyed, after a wipe or on the first attempt.

**Buffs present on Flame Leviathan during encounter which depend which tower is destroyed**:

- **Tower of Storms** - *65076*
- **Tower of Flames** - *65075*
- **Tower of Frost** - *65077*
- **Tower of Life** - *64482*

**Game Objects**
- *194370* - **Tower of Frost**
- *194371* - **Tower of Flames**
- *194375* - **Tower of Life**
- *194377* - **Tower of Storms**

----------------------------------------------
**Critical Issue:** More then 2 Demolisher Tanks are re-spawning after a wipe (like 3 or 4 more). Players are then able to take 4 Demolishers and nuke him down with Pyrite too easy. 
Other vehicles are also respawning double and they shouldn't.

There should be only only 2 and 5 vehicles of each type respawned after a wipe.
*On 10 man*
- x2 - *33109* - **[Salvaged Demolisher]**
- x2 - *33060* - **[Salvaged Siege Engine]**
- x2 - *33062* - **[Salvaged Chopper]**

*On 25 man*
- x5 - *33109* - **[Salvaged Demolisher]**
- x5 - *33060* - **[Salvaged Siege Engine]**
- x5 - *33062* - **[Salvaged Chopper]**

*I removed one demolisher in-game on 25 man mode since there were 6 and I added 3 choppers because there were only two spawned on 25.* (must correct horde / alliance version bikes in-game)

_______________________
**Achievements that should be given to a player which depend how much towers you destroyed:**
 - **Orbital Bombardment (10/25 player)** - *Defeat Flame Leviathan with at least 1 Orbital Defense System active in 10/25-player mode.* ID 10: 2913 / ID 25: 2918

 - **Orbital Devastation (10/25 player)** - *Defeat Flame Leviathan with at least 2 Orbital Defense Systems active in 10/25-player mode.* ID 10: 2914 / ID 25: 2916

 - **Nuked from Orbit (10/25 player)** - *Defeat Flame Leviathan with at least 3 Orbital Defense Systems active in 10/25-player mode.* ID 10: 2915 / ID 25: 2917

 - **Orbit-uary (10/25 player)** - *Defeat Flame Leviathan with all 4 Orbital Defense Systems active in 10/25-player mode.* ID 10: 3056 / ID 25: 3057

**Only current working achievements**: Orbit-uary (10/25 player), it works and it will pop if you activate *Secondary Defensive System* and defeat Leviathan with all 4 towers up. All other achievements will also be granted (for 1,2,3 towers up),

- **Issue achievements**: however if you destroy a tower *(you attempt Leviathan with 1-2-3 towers alive)* or any other tower, then non of the achievements for towers aren't popping, such as **Nuked from Orbit**.

*So basically what am I trying to say, if you defeat Flame Leviathan with less then 4 towers up, non of the achievements pop, and they should.*

___

*Take note If player for example achieved to kill Flame Leviathan with 3 Towers up, he will get all 3 achievements* - **Currently doesn't work**
________________
http://www.wowhead.com/achievement=2913/orbital-bombardment-10-player
http://www.wowhead.com/achievement=2918/orbital-bombardment-25-player
http://www.wowhead.com/npc=34286
http://wotlk.openwow.com/npc=33113
http://wotlk.openwow.com/npc=34003
http://www.wowhead.com/npc=33113/flame-leviathan

**Comment by Exorcista** [comment]
I was in Ulduar 25 today and i was on the Driver seat shooting my pyrite and stuff, 2nd driver graber some pyrite, i got kicked from seat, coundlnt remount and died.

**Comment by Lothloryen** [comment]
Hello there,

This was corrected time ago, thank you.
